src/engines/nwn2/script/functions_object.cpp

```diff
@@ -403,6 +403,44 @@ void Functions::getIsTrapped(Aurora::NWScript::FunctionContext &ctx) {
 	}
 }
 
+void Functions::getTrapActive(Aurora::NWScript::FunctionContext &ctx) {
+        NWN2::Object *object = NWN2::ObjectContainer::toObject(getParamObject(ctx, 0));
```

Comment:
Wrong indenting

Reply:
Sorry I'm not following your meaning. It looks to be the same indentation as the other functions.

Reply:
Nope, these there are spaces, not tabs. Look closely :P

Depending on which editor or IDE you're using, you could enable showing whitespace.
For example, I generally configure vim (and a custom git difft command) to show tabs as "→·".

![untitled](https://user-images.githubusercontent.com/156130/53256706-c4863680-36c8-11e9-9ea8-ce549531c783.png)

Reply:
Ah, okay, I see what you mean. Unfortunately the Windows copy-paste hasn't been very good at preserving tabs sometimes.